### PR TITLE
Cleanup and improve issue templates

### PR DESCRIPTION
**Pull request by bmiklautz**
* add a new template for backport issues
* fix and cleanup the bug_report template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -31,22 +31,18 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Application details**
-* Version of FreeRDP
+* FreeRDP version (xfreerdp /version)
```
Comment (akallabeth) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]: better use \`xfreerdp /version\` to have it in another font

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -31,22 +31,18 @@ A clear and concise description of what you expected to happen.
...
 * Command line used
-* output of `/buildconfig`
-* OS version connecting to
+* Output of `xfreerdp /buildconfig`
+* OS version connecting to (server side)
 * If available the log output from a run with `/log-level:trace` 
+* If you built it yourself add some notes which tag/commit/branch you have used, also your cmake parameters and
```
Comment (akallabeth) [inline .github/ISSUE_TEMPLATE/bug_report.md:39]: Maybe add `/log-level:trace 2>&1 |tee log.txt` or similar example as howto (had to answer that more than once)

**Comment by hardening** [comment]
@bmiklautz is it ok for merging ?

**Comment by hardening** [comment]
@bmiklautz @akallabeth  Is there something to change or can we merge it as is ?

**Comment by bmiklautz** [comment]
@akallabeth integrated your feedback. Sorry for the delay.

**Pull request merged**